scripts/version.sh

```diff
@@ -37,13 +37,13 @@ if [ -z "$VERSION_CONTAINERD" ]; then
     VERSION_CONTAINERD="v0.0.0"
 fi
 
-VERSION_CRICTL=$(get-module-version github.com/kubernetes-sigs/cri-tools)
+VERSION_CRICTL=$(get-module-version sigs.k8s.io/cri-tools)
 if [ -z "$VERSION_CRICTL" ]; then
     VERSION_CRICTL="v0.0.0"
 fi
 
 VERSION_K8S_K3S=$(get-module-version k8s.io/kubernetes)
-VERSION_K8S=${VERSION_K8S_K3S%"-k3s1"}
+VERSION_K8S=${VERSION_K8S_K3S%"-k3s2"}
```

Comment:
This is going to need to be undone next release when we go back to a -k3s1 of our Kubernetes fork. How about: 
```suggestion
VERSION_K8S=${VERSION_K8S_K3S%-k3s*}
```